internal/app/connectconformance/results.go

```diff
@@ -547,3 +535,118 @@ func indent(s string) string {
 	}
 	return strings.Join(lines, "\n")
 }
+
+func checkWireDetails(testCaseReq *conformancev1.ClientCompatRequest, expected, actual *conformancev1.ClientResponseResult) multiErrors {
+	expectedWire := expected.WireDetails
+	actualWire := actual.WireDetails
+	if actualWire == nil {
```

Comment:
Is it ok to bail here if just `actualWire` is nil? There could potentially be an issue where nothing is set into `actualWire` but the test case is expecting values. In that case, I think we'd miss that failure right?

Reply:
Only the reference client populates it. So it will be nil when running tests in client mode (i.e. testing some other client), and that's definitely okay, even if the test case has an "expected" value.

Also, the only thing we should ever put in a test case in here is the status code. Everything else is indirectly checked by the reference client already (since client already processes the connect errors, trailers, etc). So _if_ they are here, then we check them with extra strictness.

Part of me was actually wondering if we should get rid of the wire details and just do all of these checks directly inside the reference client (instead of here in the test runner assertions), and have the client send back any errors via the feedback field. If we did that, the only field we need in the response is the actual status code...